### PR TITLE
guide: edits around `gdrive_acknowledge_abuse`

### PR DESCRIPTION
**Pull request by jorgeorpinel**
Follow-up to #4187

Rel. #4189

### CODE REVIEW

File: content/docs/user-guide/how-to/setup-google-drive-remote.md [hunk 1]
```diff
@@ -248,12 +251,6 @@ $ dvc remote modify myremote gdrive_acknowledge_abuse true
 </admon>
 > Please note our [Privacy Policy (Google APIs)](/doc/user-guide/privacy).
```
Comment (dberenbaum) [inline content/docs/user-guide/how-to/setup-google-drive-remote.md:252]: Unrelated, but this is broken. Where is it supposed to link to/do we need it?

**Review by aliahari — APPROVED** [review]

File: content/docs/user-guide/how-to/setup-google-drive-remote.md (hunk 1 above)
Comment (jorgeorpinel) [reply content/docs/user-guide/how-to/setup-google-drive-remote.md:252]: It's not broken 🙂 just not linked from nav: 

https://dvc.org/doc/user-guide/privacy

Comment (dberenbaum) [reply content/docs/user-guide/how-to/setup-google-drive-remote.md:252]:
```suggestion
> Please note our [Privacy Policy (Google APIs)](https://dvc.org/doc/user-guide/privacy).
```

So how about this?

Comment (jorgeorpinel) [reply content/docs/user-guide/how-to/setup-google-drive-remote.md:252]: Ooooh I see what you mean. It does not render correctly it seems. Interesting... OK committing, thanks.

**Review by dberenbaum — APPROVED** [review]
Thanks @jorgeorpinel!

**Pull request merged**